doc/build-aux/pandoc-filters/docbook-writer/html-elements.lua

```diff
@@ -0,0 +1,11 @@
+--[[
+Converts some HTML elements commonly used in Markdown to corresponding DocBook elements.
+]]
+
+function RawInline(elem)
+  if elem.format == 'html' and elem.text == '<kbd>' then
+    return pandoc.RawInline('docbook', '<keycap>')
+  elseif elem.format == 'html' and elem.text == '</kbd>' then
+    return pandoc.RawInline('docbook', '</keycap>')
+  end
+end
```

Comment:
Thats handy! We might also extract that and merge this already.

Reply:
@jtojnar contributed it in https://github.com/davidak/nixpkgs/pull/1 :)

i hope merging this PR does not take too long